tests/components/FilterSearch.stories.tsx

```diff
@@ -90,11 +90,12 @@ DropdownHighlight.parameters = {
     autoCompleteService: generateMockedAutocompleteService(undefined, labeledFilterSearchResponse)
   }
 };
-DropdownHighlight.play = ({ canvasElement }) => {
+DropdownHighlight.play = async ({ canvasElement }) => {
```

Comment:
I think we can just delete this story now since the other 2 stories with expanded dropdowns will already show what a highlighted value looks like